WordPress/Classes/Utility/WPCrashlytics.h

```diff
@@ -8,6 +8,24 @@
  */
 @interface WPCrashlytics : NSObject
 
+#pragma mark - Init
+
 - (instancetype)initWithAPIKey:(NSString *)apiKey;
 
+#pragma mark - User Opt Out
+
+/**
+ *  @brief      Call this method to know if the user has opted out of crashalytics tracking.
```

Comment:
Suuuuper nitpicky – `crashalytics` -> `Crashlytics`

Reply:
Gah!. You are still my human spellchecker @frosty !! 😆 👍